platform/platform-scheduler/src/test/java/ddf/platform/scheduler/TestCommandJob.java

```diff
@@ -152,6 +153,21 @@ public void testSimpleCommand() throws Exception {
 
     }
 
+    @Test
+    public void testNullContext() throws JobExecutionException {
+        CommandJob job = getCommandJob();
+        job.doExecute(null);
+    }
+
+    @Test
+    public void testNullMergedJobDataMap() throws JobExecutionException {
+        CommandJob job = getCommandJob();
+
+        //explicitly not mocking getMergedJobDataMap()
```

Comment:
Instead of a comment, should we do `when(jobExecutionContext.getMergedJobDataMap()).thenReturn(null);`?